pom.xml

```diff
@@ -73,30 +73,40 @@
             <version>3.1</version>
         </dependency>
         <dependency>
+            <groupId>com.github.tgalery</groupId>
+            <artifactId>dkpro-jwpl</artifactId>
+            <version>53c5866b70</version>
+        </dependency>
+        <!-- <dependency>
             <groupId>de.tudarmstadt.ukp.wikipedia</groupId>
             <artifactId>de.tudarmstadt.ukp.wikipedia.api</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.1-SNAPSHOT</version>
```

Comment:
No need to change the version in a commented out dependency (and I'd have [something to say](https://github.com/idio/json-wikipedia/pull/49#issuecomment-337179877) about depending on snapshot versions anyway).

I'd prefer this was deleted altogether, that's what version control is for.